File: Sim06-EsameInClasse/src/Furgone.java
public class Furgone extends Veicolo {
    Boolean rimorchio;
    int numero_posti;
    Boolean vanoDiCarico;
    public Furgone(int codice, String tipo, String targa, String modello, String marca, Boolean rimorchio,
            int numero_posti, Boolean vanoDiCarico) {
        super(codice, tipo, targa, modello, marca);
        this.rimorchio = rimorchio;
        this.numero_posti = numero_posti;
        this.vanoDiCarico = vanoDiCarico;
    }
    public Boolean getRimorchio() {
        return rimorchio;
    }
    public int getNumero_posti() {
        return numero_posti;
    }
    public Boolean getVanoDiCarico() {
        return vanoDiCarico;
    }

//agagliaio ,	cilindrata, categoria

//"tipo, targa, codice, modello, marca, costo giornaliero,bagagliaio, cilindrata, categoria, rimorchio, numero di posti,vano di carico"
public String toString(){
    return "Furgone  "+targa+codice+modello+marca+costo_giornaliero+"- "+" - "+" - "+rimorchio+numero_posti+vanoDiCarico;
}
}

File: 40/Furgone.py
from Veicoli import Veicoli

class Furgone(Veicoli):
    def __init__(self, codice, tipo, targa, modello, marca, costoGiornaliero,categoria,numPosti,vanoDiCarico):
        super().__init__(codice, tipo, targa, modello, marca, costoGiornaliero)
        self.categoria = categoria
        self.numPosti  = numPosti
        self.vanoDiCarico = vanoDiCarico
    

    "tipo, targa, codice, modello, marca, costo giornaliero,cilindrata, bagagliaio, categoria, numero di posti, vano di carico"
    def __str__(self) -> str:
        return self.tipo+"\t"+self.targa+"\t"+str(self.codice)+"\t"+self.modello+"\t"+self.marca+"\t"+str(self.costoGiornaliero)+"\t-"+"\t-"+self.categoria+"\t"+str(self.numPosti)+"\t"+str(self.vanoDiCarico)